packages/cli/src/evaluation.ee/test-runner/__tests__/get-start-node.ee.test.ts

```diff
@@ -1,40 +0,0 @@
-import { readFileSync } from 'fs';
-import path from 'path';
-
-import { getPastExecutionTriggerNode } from '../utils.ee';
-
-const executionDataJson = JSON.parse(
-	readFileSync(path.join(__dirname, './mock-data/execution-data.json'), { encoding: 'utf-8' }),
-);
-
-const executionDataMultipleTriggersJson = JSON.parse(
-	readFileSync(path.join(__dirname, './mock-data/execution-data.multiple-triggers.json'), {
-		encoding: 'utf-8',
-	}),
-);
-
-const executionDataMultipleTriggersJson2 = JSON.parse(
-	readFileSync(path.join(__dirname, './mock-data/execution-data.multiple-triggers-2.json'), {
-		encoding: 'utf-8',
-	}),
-);
-
-describe('getPastExecutionStartNode', () => {
-	test('should return the start node of the past execution', () => {
-		const startNode = getPastExecutionTriggerNode(executionDataJson);
```

Comment:
should we delete these tests if the utility function `getPastExecutionTriggerNode` still exists?

Reply:
`getPastExecutionTriggerNode` is not used anymore, deleted `utils` completely